arrocke/gloss-translation
# feat: updated LSJ entries

## What has changed


- Added data source section to `README`
- Updated greek data file
- Updated LSJ import logic to support the new data file
- Modified BDB import logic
- Updated seed file

## Connected Issues


Closes #277 

## QA Steps

- `nx run db:import-lsj`
- In database: `SELECT * FROM "LemmaResource" WHERE "resourceCode" = 'LSJ';`
    - NOTE: the definitions are multi-line. Click on one to see the whole entry.



## Post-Deployment



- [ ] Run `nx run db:import-db` for production DB
